documentation/modules/security/con-certificate-renewal.adoc

```diff
@@ -44,21 +43,21 @@ This is not strictly necessary because the signing key has not changed, but it k
 
 . Restart the Topic and User Operators so that they will trust the new CA certificate and use the new client certificates.
 
-
 == Client applications
 
-The Cluster Operator is not aware of all the client applications using the Kafka cluster.
-
-IMPORTANT: Depending on how your applications are configured, you might need take action to ensure they continue working after certificate renewal.
-
-Consider the following important points to ensure that client applications continue working.
+The Cluster Operator is not aware of the client applications using the Kafka cluster.
 
-* When they connect to the cluster, client applications must trust the cluster CA certificate published in _<cluster>_-cluster-ca-cert.
+When connecting to the cluster, and to ensure they operate correctly, client applications must:
 
-* When using the User Operator to provision client certificates, client applications must use the current `user.crt` and `user.key` published in their `_<user>_` `Secret` when they connect to the cluster.
-For workloads running inside the same Kubernetes cluster this can be achieved by mounting the secrets as a volume and having the client Pods construct their key- and truststores from the current state of the `Secrets`. 
-For more details on this procedure, see xref:configuring-internal-clients-to-trust-cluster-ca-{context}[].
+* Trust the cluster CA certificate published in the _<cluster>_-cluster-ca-cert Secret.
+* Use the current `user.p12` and `user.password`, or `user.crt` and `user.key`, published in their _<user-name>_ Secret to connect to the cluster, when using the User Operator to provision client certificates.
```

Comment:
This sentence is quite difficult to parse. Could it be broken down?

Reply:
True.
I don't think we need to specify the fields here, as we decribe them later in teh section. So I've simplified to:

* Use the credentials published in their _<user-name>_ Secret to connect to the cluster.
+
The User Secret provides credentials in PEM and PKCS #12 formats.
The User Operator creates the user credentials when a user is created.